projects/canopy/src/lib/card/docs/promotions/guide.stories.mdx

```diff
@@ -0,0 +1,513 @@
+        <lg-card-hero-img>
+          <lg-brand-icon [name]="iconName" size="xs"></lg-brand-icon>
+        </lg-card-hero-img>
+        <lg-card-content>
+          <h3 lgMarginBottom="xxs" class="lg-font--expressive">{{ title }}</h3>
+          <p lgMarginBottom="lg">{{ text }}</p>
+          <a href="#">{{ buttonText }}</a>
+        </lg-card-content>
+      </lg-card>
+    </div>
+    <div lgColSm="12" lgColLg="4">
+      <lg-card
+        lgShadow
+        lgMarginBottom="lg"
+        variant="promotion"
+        [orientation]="{ sm: 'vertical', md: 'horizontal', lg: 'vertical' }">
+        <lg-card-hero-img>
+          <lg-brand-icon [name]="iconName" size="xs"></lg-brand-icon>
+        </lg-card-hero-img>
+        <lg-card-content>
+          <h3 lgMarginBottom="xxs" class="lg-font--expressive">{{ title }}</h3>
+          <p lgMarginBottom="lg">{{ text }}</p>
+          <a href="#">{{ buttonText }}</a>
+        </lg-card-content>
+      </lg-card>
+    </div>
+  </div>
+  <div lgRow>
+    <div lgCol="12">
+      <lg-separator variant="dotted" lgMarginTop="none"></lg-separator>
+    </div>
+    <div lgColSm="12" lgColLg="6">
+      <lg-card
+        lgShadow
+        lgMarginBottom="lg"
+        variant="promotion"
+        [orientation]="{ sm: 'vertical', md: 'horizontal', lg: 'vertical' }">
+        <lg-card-hero-img>
+          <lg-brand-icon [name]="iconName" size="xs"></lg-brand-icon>
+        </lg-card-hero-img>
+        <lg-card-content>
+          <h3 lgMarginBottom="xxs" class="lg-font--expressive">{{ title }}</h3>
+          <p lgMarginBottom="lg">{{ text }}</p>
+          <a href="#">{{ buttonText }}</a>
+        </lg-card-content>
+      </lg-card>
+    </div>
+    <div lgColSm="12" lgColLg="6">
+      <lg-card
+        lgShadow
+        lgMarginBottom="lg"
+        variant="promotion"
+        [orientation]="{ sm: 'vertical', md: 'horizontal', lg: 'vertical' }">
+        <lg-card-hero-img>
+          <lg-brand-icon [name]="iconName" size="xs"></lg-brand-icon>
+        </lg-card-hero-img>
+        <lg-card-content>
+          <h3 lgMarginBottom="xxs" class="lg-font--expressive">{{ title }}</h3>
+          <p lgMarginBottom="lg">{{ text }}</p>
+          <a href="#">{{ buttonText }}</a>
+        </lg-card-content>
+      </lg-card>
+    </div>
+  </div>
+</div>
+```
+## Additional development details
+
+### LgCardComponent
+
+The main body of the promotion card. It's composed by the following components:
+
+- `LgCardHeroImageComponent`
+- `LgCardContent`
+
+#### Inputs
+The orientation input arranges the direct children of the card in vertical or horizontal layout, a responsive way.
+It accepts an object, where the **key** is a mobile-first breakpoint from the standard list of available breakpoints: `xs`, `sm`, `md`, `lg`, `xl`, `xxl`.
+The value is from the set of available Orientation options.
+The current available options for Orientation are `vertical`, `horizontal`, `reverse-vertical`, `reverse-horizontal`.
```

Comment:
Could we add a paragraph to explain what `reverse-vertical` and `reverse-horizontal` do? not sure it's clear enough.

Reply:
Done in the Orientation Directive Guide. :)